docs/spec/images/overview-reference.dot

```diff
@@ -0,0 +1,32 @@
+// This is the overview graph for object refrences in the control plane.
```

Comment:
This graph does not include Flows at all.

Reply:
I added a Flow, but flow is flexible so it is hard to represent in a single graph